home/home.js

```diff
@@ -0,0 +1,41 @@
+document.getElementById("embed").innerHTML = '<iframe src="https://accio1.github.io/SSE/#FEATURED" style="width: 250px; min-height: 538.5px; border: none;"></iframe>';
+document.getElementById("Ecode").innerHTML = '<iframe src="https://accio1.github.io/SSE/#FEATURED" style="width: 250px; min-height: 538.5px; border: none;"></iframe>';
+	
+function GoTo() {
+	var GT_StudioID;
+	GT_StudioID = document.getElementById("GT_StudioID").value;
+		if (isNaN(GT_StudioID) && GT_StudioID !=="SDS" && GT_StudioID !=="FEATURED") {
+			document.getElementById("GT_result").innerHTML = "Not A Studio ID";
+  			} 
+		else {
+    			window.location.href = "/SSE/#"+GT_StudioID;
```

Comment:
Use `..` instead of `/SSE` (for testing locally or with Netlify)

Reply:
Done